src/__tests__/shared/components/navbar/Navbar.test.tsx

```diff
@@ -101,107 +103,80 @@ describe('Navbar', () => {
+    describe('header', () => {
+      it('should render a HospitalRun Navbar', () => {
+        setup(allPermissions)
 
-    expect(hospitalRunNavbar).toHaveLength(1)
-  })
-
-  describe('header', () => {
-    it('should render a HospitalRun Navbar with the navbar header', () => {
-      const wrapper = setup(allPermissions)
-      const header = wrapper.find('.nav-header')
-      const { children } = header.first().props() as any
+        expect(screen.getByText(/hospitalrun/i)).toBeInTheDocument()
+        expect(screen.getByRole('button')).toBeInTheDocument()
+      })
 
-      expect(children.props.children).toEqual('HospitalRun')
-    })
+      it('should navigate to / when the header is clicked', () => {
+        setup(allPermissions)
 
-    it('should navigate to / when the header is clicked', () => {
-      const wrapper = setup(allPermissions)
-      const header = wrapper.find('.nav-header')
+        history.location.pathname = '/enterprise-1701'
+        expect(history.location.pathname).not.toEqual('/')
+        userEvent.click(screen.getByText(/hospitalrun/i))
 
-      act(() => {
-        const onClick = header.first().prop('onClick') as any
-        onClick()
+        expect(history.location.pathname).toEqual('/')
       })
-
-      expect(history.location.pathname).toEqual('/')
     })
-  })
-
-  describe('add new', () => {
-    it('should show a shortcut if user has a permission', () => {
-      const wrapper = setup(allPermissions)
-      const hospitalRunNavbar = wrapper.find(HospitalRunNavbar)
-      const addNew = hospitalRunNavbar.find('.nav-add-new')
-      const { children } = addNew.first().props() as any
 
-      const [icon, label] = children[0].props.children
+    describe('add new', () => {
+      it('should show a shortcut if user has a permission', () => {
+        setup(allPermissions)
 
-      expect(icon.props.icon).toEqual('patient-add')
-      expect(label).toEqual('patients.newPatient')
-    })
+        const navButton = screen.getByRole('button')
+        userEvent.click(navButton)
+        expect(
+          screen.getByRole('button', {
+            name: /patients\.newpatient/i,
+          }),
+        ).toBeInTheDocument()
 
-    it('should not show a shortcut if user does not have a permission', () => {
-      // exclude labs and incidents permissions
-      const wrapper = setup(cloneDeep(allPermissions).slice(0, 6))
-      const hospitalRunNavbar = wrapper.find(HospitalRunNavbar)
-      const addNew = hospitalRunNavbar.find('.nav-add-new')
-      const { children } = addNew.first().props() as any
-
-      children.forEach((option: any) => {
-        expect(option.props.children).not.toEqual('labs.requests.new')
-        expect(option.props.children).not.toEqual('incidents.requests.new')
-        expect(option.props.children).not.toEqual('imagings.requests.new')
+        // 0 & 1 index are dashboard fixed elements, 2 index is first menu label for user
```

Comment:
so if those elements are removed, this test would fail. Is that desired behaviour?

Reply:
Not sure what you are asking about here. The index of 2 is the first label that the user actually interacts with meaningfully, which is the point of the test "shortcut" should be the first element the user wants from the permissions available.